### PR TITLE
Added another parsing phrase to QChem loadEnergy()

### PR DESCRIPTION
**Pull request by alongd**
Added another parsing phrase in Arkane for QChem log files.
Parsing a line in the following format:
` Total energy in the final basis set = -328.6420045839`
Also some pythonic modifications to this block.
An input file for example and review of the code is attached.
[output.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/2660465/output.txt)

### CODE REVIEW

File: arkane/qchem.py [hunk 1]
```diff
@@ -270,29 +270,20 @@ def loadEnergy(self, frequencyScaleFactor=1.):
...
-#                ZPE = float(line.split()[4]) * 4184
-#                scaledZPE = ZPE * frequencyScaleFactor
-#                print 'ZPE is ' + str(ZPE)
-            # Read the next line in the file
+        with open(self.path, 'r') as f:
             line = f.readline()
-    
-        # Close file when finished
-        f.close()
+            while line != '':
```
Comment (mliu49) [inline arkane/qchem.py]: While you're making it more pythonic, the following is a lot simpler if you're going to read the whole file:
```
with open(self.path, 'r') as f:
    for line in f:
        # Do something
```

Comment (alongd) [reply arkane/qchem.py]: Thanks for the comment!

**Comment by alongd** [comment]
@rgillis8, this branch now passes all tests, I also incorporated @mliu49's suggestion. All of Arkane's examples were run on this branch and terminated successfully. I think this PR should be ready for a final review, whenever you got time. Thanks!

**Comment by alongd** [comment]
@rgillis8, I added a minor change (lines 277-281), splitting the two checks into different loops, giving priority to `Final energy is` over `Total energy in the final basis set`.
(This matters, for example, if a freq job is executed after the sp job in the same run)
Could you take a final look? Thanks!

**Comment by alongd** [comment]
Thanks @rgillis8!
I just rebased, I think it needs an additional approval after the tests are done

**Pull request merged**